src/client/app/Sidebar/RightSidebar.js

```diff
@@ -71,7 +71,7 @@ export default class RightSidebar extends React.Component {
       <div>
         {!authenticated && <SignUp />}
         <Switch>
-          <Route path="/activity" render={() => <UserActivitySearch />} />
+          <Route path="/activity" render={() => <UserActivitySearch isCurrentUser />} />
```

Comment:
We can use `component` instead of `render` and then we don't need `withRouter`.

Reply:
good call, just updated